src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/actions/PlayActions.kt

```diff
@@ -198,4 +202,22 @@ class SingleStepInUnrealAction : PlayStateAction(
         state.disableUntilStateChange()
         host.model.requestFrameSkipFromRider.fire(state.nextRequestID())
     }
+}
+
+class RefreshProjects : DumbAwareAction(AllIcons.Actions.Refresh) {
+    override fun update(e: AnActionEvent) {
+        super.update(e)
+        val host = e.getHost()
+        e.presentation.isVisible = host?.isUnrealEngineSolution ?: false
+        e.presentation.isEnabled = host?.isConnectedToUnrealEditor ?: false
```

Comment:
Why it is enabled only when Rider is connected to Unreal Editor?
Also is it true that `isUnrealEngineSolution` will be false when uproject is opened?

Reply:
1. Ok, it seems to be wrong
2. No. Is there any potential problems, if this button will be enabled in uproject?

Reply:
2. It won't work there so I guess it doesn't make sense to make it visible for users :)